userguide/tutorials/catalog-examples.adoc

```diff
@@ -659,6 +659,88 @@ This is the default rule. It specifies that any other plan changes should occur
 |Trigger invoice dry run with target date `2021-10-29`    |Invoice generated for  `$100` for `2021-10-29` to `2021-11-29` corresponding to the `standard-monthly` plan
 |===
 
+== Usage Billing
+
+Kill Bill supports usage based billing as explained in the https://docs.killbill.io/latest/userguide_subscription.html#components-catalog-usage[_Subscription Billing_] document. A quick summary about usage billing:
+
+* Users are billed based on their usage data as defined in the catalog. The `Plan` section in the catalog can contain one or more usage section(s) in addition to other non usage based sections.
+
+* Kill Bill supports two types of usage billing: `CONSUMABLE` and `CAPACITY`
+
+* In `CONSUMABLE` billing, users are billed based on the number of units consumed. Two `TierBlockPolicy` policies are supported: `ALL_TIER` and `TOP_TIER`. The `ALL_TIER` policy charges users across each tier while the `TOP_TIER` policy charges users at the top tier.
+
+* In `CAPACITY` billing users are charged a flat rate based on the units consumed.
+
+Let us take a look at a few catalog examples. 
+
+=== Consumable Billing with ALL_TIERS policy
+
+This example uses the https://github.com/killbill/killbill-docs/tree/4671dcd9da1cf021e85629ab67e3ffb6fb553bb1/catalogs/consumable-billing-all-tiers.xml[Consumable Billing All Tiers] catalog. It has a `water-monthly` plan with a `MONTHLY` recurring charge of `$30`. In addition, it also has a usage section with `usageType=CONSUMABLE`, `tierBlockPolicy=ALL_TIERS` and the following `tieredBlock` sections:
```

Comment:
@reshmabidikar Heads up: some of the links don't seem to work

Reply:
Fixed and created [this PR](https://github.com/killbill/killbill-docs/pull/360)